From c40ea3d9a3be8645441967cddf14645b84af5f2c Mon Sep 17 00:00:00 2001
From: Andreas Jaeger <aj@suse.de>
Date: Tue, 8 Jan 2013 19:32:00 +0100
Subject: [PATCH] BZ#14985: Remove erroneous EPOLL_NONBLOCK

	[BZ# 14985]
	* sysdeps/unix/sysv/linux/sparc/bits/epoll.h (EPOLL_NONBLOCK):
	Remove.
	* sysdeps/unix/sysv/linux/bits/epoll.h (EPOLL_NONBLOCK):
	Likewise.
	* sysdeps/unix/sysv/linux/x86/bits/epoll.h (EPOLL_NONBLOCK):
	Likewise.

ChangeLog.mips:
	[BZ# 14985]
	* sysdeps/unix/sysv/linux/mips/bits/epoll.h (EPOLL_NONBLOCK):
	* Remove.

ChangeLog.hppa:
	[BZ# 14985]
	* sysdeps/unix/sysv/linux/hppa/sys/epoll.h (EPOLL_NONBLOCK):
	Remove.

ChangeLog.alpha:
	[BZ# 14985]
	* sysdeps/unix/sysv/linux/alpha/bits/epoll.h (EPOLL_NONBLOCK):
	Remove.
---
 sysdeps/unix/sysv/linux/bits/epoll.h             | 4 +---
 sysdeps/unix/sysv/linux/sparc/bits/epoll.h       | 4 +---
 sysdeps/unix/sysv/linux/x86/bits/epoll.h         | 4 +---
 11 files changed, 31 insertions(+), 19 deletions(-)

diff --git a/sysdeps/unix/sysv/linux/bits/epoll.h b/sysdeps/unix/sysv/linux/bits/epoll.h
index 21b40d3..d8b8205 100644
--- a/sysdeps/unix/sysv/linux/bits/epoll.h
+++ b/sysdeps/unix/sysv/linux/bits/epoll.h
@@ -22,8 +22,6 @@
 /* Flags to be passed to epoll_create1.  */
 enum
   {
-    EPOLL_CLOEXEC = 02000000,
+    EPOLL_CLOEXEC = 02000000
 #define EPOLL_CLOEXEC EPOLL_CLOEXEC
-    EPOLL_NONBLOCK = 00004000
-#define EPOLL_NONBLOCK EPOLL_NONBLOCK
   };
diff --git a/sysdeps/unix/sysv/linux/sparc/bits/epoll.h b/sysdeps/unix/sysv/linux/sparc/bits/epoll.h
index 8c9968e..d984723 100644
--- a/sysdeps/unix/sysv/linux/sparc/bits/epoll.h
+++ b/sysdeps/unix/sysv/linux/sparc/bits/epoll.h
@@ -22,8 +22,6 @@
 /* Flags to be passed to epoll_create1.  */
 enum
   {
-    EPOLL_CLOEXEC = 0x400000,
+    EPOLL_CLOEXEC = 0x400000
 #define EPOLL_CLOEXEC EPOLL_CLOEXEC
-    EPOLL_NONBLOCK = 0x004000
-#define EPOLL_NONBLOCK EPOLL_NONBLOCK
   };
diff --git a/sysdeps/unix/sysv/linux/x86/bits/epoll.h b/sysdeps/unix/sysv/linux/x86/bits/epoll.h
index 479645e..de908bc 100644
--- a/sysdeps/unix/sysv/linux/x86/bits/epoll.h
+++ b/sysdeps/unix/sysv/linux/x86/bits/epoll.h
@@ -22,10 +22,8 @@
 /* Flags to be passed to epoll_create1.  */
 enum
   {
-    EPOLL_CLOEXEC = 02000000,
+    EPOLL_CLOEXEC = 02000000
 #define EPOLL_CLOEXEC EPOLL_CLOEXEC
-    EPOLL_NONBLOCK = 00004000
-#define EPOLL_NONBLOCK EPOLL_NONBLOCK
   };
 
 #define __EPOLL_PACKED __attribute__ ((__packed__))
-- 
1.8.1.2

